File: scratch/read_excel.js
const XLSX = require('xlsx');
try {
  const wb = XLSX.readFile('C:\\Users\\mohdm\\Downloads\\FMAC_Requests_All_2026-05-18.xlsx', { cellDates: true });
  const ws = wb.Sheets[wb.SheetNames[0]];
  const data = XLSX.utils.sheet_to_json(ws);
  if (data.length > 0) {
    console.log("Columns:", Object.keys(data[0]));
    console.log("First row:", data[0]);
  } else {
    console.log("Empty sheet.");
  }
} catch (e) {
  console.error("Error:", e);
}

File: scratch/read_excel.py
import pandas as pd
import json

try:
    df = pd.read_excel(r"C:\Users\mohdm\Downloads\FMAC_Requests_All_2026-05-18.xlsx")
    print("Columns:", df.columns.tolist())
    print("\nFirst 2 rows:")
    print(json.dumps(df.head(2).to_dict(orient='records'), default=str, indent=2))
except Exception as e:
    print("Error:", e)
